deployment/roles/deploy/templates/log-files-pvc.yml.j2

```diff
@@ -0,0 +1,14 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: log-files-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
```

Comment:
The question is whether `RWX` mode is actually supported by the cluster we're about to deploy to.
Can you try first?

Reply:
yes, I will try

Reply:
I checked in the web ui and there was an option RWX available when I opened PVC creation options:
```
Access Mode
Single User (RWO) Shared Access (RWX) Read Only (ROX)

```

Reply:
Ok, it should work then.
But I'd still suggest to try to create a PVC from your template before you merge this - just to be sure it doesn't spit out any errors later.

Reply:
I understand, I deployed just the PVC and it went well.